src/View/Helper/FlashHelper.php

```diff
@@ -26,7 +26,16 @@ class FlashHelper extends Helper
 {
 
     /**
-     * Used to render the message set in FlashComponent::set()
+     * Default config for the helper.
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'stackElement' => false
```

Comment:
I don't really understand the question. Why should they share data? This config option is a string that contain the path to an optionnal element in which the stack of messages can be rendered